data/virtualization/autoyast/host_15.xml.ep

```diff
@@ -165,7 +167,7 @@
   <login_settings/>
   <networking>
     <keep_install_network config:type="boolean">true</keep_install_network>
-    % if ($get_var->('KUBEVIRT_TEST') && $get_var->('WORKER_CLASS') =~ /virt-mm-baremetal-ipmi/) {
+    % if ($get_var->('KUBEVIRT_TEST') || $get_var->('WORKER_CLASS') =~ /virt-mm-baremetal-ipmi/) {
```

Comment:
We have much more machines now, both uefi and non-uefi. So using worker_class here may not be widely applicable. What's the essential requirements for this line? We'd better consider replacing it.

Reply:
There was issue about using `<bridge_ports>eth0 eth1</bridge_ports>` that disrupted openQA. 

I think but have to run tests, that we can switch all sle15 test to use the new setup `    <virt_bridge_proposal config:type="boolean">true</virt_bridge_proposal>` for uefi and non-uefi.